inst/shinys/multicollinearity/app.R

```diff
@@ -8,15 +8,15 @@ ui <- fluidPage(
   br(),
   fluidRow(
 
-    column(width = 3,
+    column(width = 4,
            sliderInput("corr_x1x2", "Correlation(x1, x2)", min = 0,
                        max = 1, step = .01, value = 0),
            br(),
            br(),
 
-           textOutput("standarderrors")),
+           verbatimTextOutput("standarderrors")),
 
-    column(width = 9,
+    column(width = 8,
            plotlyOutput("planefit"))
 
 ))
```

Comment:
it would be good to have a bigger impact on R2:
* more points? N=300
* smaller variance on outcome? rnorm(0,1)?